### ISSUE TITLE
reference Bert

### ISSUE THREAD

**Issue by chanzae** [chanzae/NLP_study#1]
1. 서브워드 구축
https://keep-steady.tistory.com/37

2. BERT with MECAB tokenizer
https://github.com/yeontaek/BERT-MECAB-Korean-Model

3. BERT 관련 링크 모음집
https://cleancode-ws.tistory.com/103

4. 데이콘 뉴스토픽 분류 베이스 코드
https://dacon.io/competitions/official/235747/codeshare/2905

5. 데이콘 BERT 베이스 코드
https://dacon.io/codeshare/2861

6. BERT 기초
https://han-py.tistory.com/267

7. TensorFlow와 Transformer를 활용한 BERT 베이스
https://velog.io/@jaehyeong/Fine-tuning-Bert-using-Transformers-and-TensorFlow

8. KoBERT 다중분류 with 파이토치 
https://velog.io/@seolini43/KOBERT%EB%A1%9C-%EB%8B%A4%EC%A4%91-%EB%B6%84%EB%A5%98-%EB%AA%A8%EB%8D%B8-%EB%A7%8C%EB%93%A4%EA%B8%B0-%ED%8C%8C%EC%9D%B4%EC%8D%ACColab

**Comment by chanzae** [comment]
chane label
